tests/test_caching.py

```diff
@@ -112,7 +120,9 @@ def test_non_exisiting_directory(parent_directory):
         assert len(cache.cached_responses) == 1
         assert cache.get_cached_response(request) == response
     # Delete newly-created cache file and directory
-    os.remove(os.path.join(cache_dir, "sample_cache.sqlite"))
+    os.remove(
+        os.path.join(cache_dir, "0097de8ba7e8d5c79d93cf4e81926c5bfe77bd1f.sqlite")
```

Comment:
I think there is some advantage to the output files having human readable names. For example, someone could go and delete the cache for a specific SUT.

What do you think about introducing a `normalized_filename` method that replaces `/` with either `_` or `-`?

Reply:
Sounds like there are not any concerns about other special characters. Update to `normalized_filename` method with a replace. Thanks.